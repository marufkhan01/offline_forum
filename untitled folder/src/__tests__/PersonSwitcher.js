import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import PersonaSwitcher from '../components/PersonaSwitcher';


it('PersonaSwitcher component should render as expected',() =>{
    const component = shallow(<PersonaSwitcher />);
    const tree = toJSON(component);
    expect(tree).toMatchSnapshot();
});