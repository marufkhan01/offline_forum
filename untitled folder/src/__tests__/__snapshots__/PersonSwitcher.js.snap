// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonaSwitcher component should render as expected 1`] = `
<div
  className="fixed pin-t pin-r"
>
  <AvatarSelector />
  <div
    className="inline-block relative p-4"
  >
    <select
      className="block appearance-none bg-white border border-grey-light hover:border-grey px-4 py-2 pr-8 rounded shadow"
    >
      <option
        value="Zac"
      >
        Zac
      </option>
      <option
        value="Esmeralda"
      >
        Esmeralda
      </option>
      <option
        value="Morgana"
      >
        Morgana
      </option>
    </select>
    <div
      className="pointer-events-none absolute pin-y pin-r flex items-center px-6 text-grey-darker"
    >
      <svg
        className="fill-current h-4 w-4"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z"
        />
      </svg>
    </div>
  </div>
</div>
`;
